templates/boltzmann.js

```diff
@@ -2150,6 +2150,7 @@ function session ({
   cookie = process.env.SESSION_ID || 'sid',
   secret = process.env.SESSION_SECRET,
   salt = process.env.SESSION_SALT,
+  logger = bole('BOLTZMANN:session'),
```

Comment:
Elsewhere we've lowercased `boltzmann` in logs. (I'm not opposed to uppercasing it elsewhere, but we should probably pick one case)